Rakefile

```diff
@@ -1,4 +1 @@
-require "bundler/gem_tasks"
```

Comment:
Are you sure about removing this line? Without it we no longer have `rake build`, `rake install`, and a few other rake tasks.

Reply:
No, I don't want to break that. Thanks!